inspec.yml

```diff
@@ -0,0 +1,88 @@
+name: cms-ars-5.0-oracle-database-19c-cis-overlay
+title: Oracle Database 19c based on 19c CIS Benchmark
+maintainer: MITRE SAF Team
+copyright: MITRE, 2022
+copyright_email: saf@groups.mitre.org
+license: Apache-2.0
+summary: InSpec testing for overlay compliance with the Oracle 19c
+inspec_version: ">= 4.0"
+version: 1.0.0
+
+depends:
+  - name: oracle-database-19c-cis-baseline
+    url: https://github.com/mitre/oracle-database-19c-cis-baseline/archive/main.tar.gz
+
+inputs:
+  - name: overlay_controls
+    desc: 'List of inapplicable controls/requirements in ARS 5.0 based on the system categorization'
+    type: Array
+    <% if ENV['BASELINE'] == 'High-HVA' %>
+    value: []
+    <% elsif ENV['BASELINE'] == 'High' %>
+    value: []
+    <% elsif ENV['BASELINE'] == 'Moderate-HVA' %>
+    value: []
+    <% elsif ENV['BASELINE'] == 'Moderate' || ENV['BASELINE'].nil? %> # Default
+    value: 
+      - "oracle19c-6.1.12" # CM-6 (2)
+      - "oracle19c-6.1.3"  # CM-6 (2)
+    <% elsif ENV['BASELINE'] == 'Low-HVA' %>
+    value: []
+    <% elsif ENV['BASELINE'] == 'Low' %>
+    value:
+      - "oracle19c-2.1.1" # AC-6 
+      - "oracle19c-2.2.11" # AC-6 
+      - "oracle19c-2.2.14" # AC-6 
+      - "oracle19c-3.8" # AC-6 
+      - "oracle19c-3.9" # AC-6 
+      - "oracle19c-4.2" # AC-6 
+      - "oracle19c-5.1.2.1" # AC-6 
+      - "oracle19c-4.1" # CM-2 (2)
+      - "oracle19c-6.1.12" # CM-6 (2)
+      - "oracle19c-6.1.3" # CM-6 (2)
+      - "oracle19c-2.2.15" # SC - 8
+      - "oracle19c-2.2.16" # SC - 8
+      - "oracle19c-5.2.1" # SC - 8
+      - "oracle19c-5.2.5" # SC - 8
+      - "oracle19c-5.2.6" # SC - 8
+      - "oracle19c-5.2.8" # SC - 8
+      - "oracle19c-4.5" # SC-28
+    <% end %>
+
+# oracle19c-2.2.10
+  - name: sec_max_failed_login_attempts
+    description: 'Number of maximum failed login attempts, in seconds'
+    type: Numeric
+    <% if ENV['BASELINE'] == 'High' %>
+    value: 3
+    <% elsif ENV['BASELINE'] == 'Moderate' || ENV['BASELINE'].nil? || ENV['BASELINE'] == 'Low' %> # Default
+    value: 5
+    <% end %>
+    profile: oracle-database-19c-cis-baseline
+
+# oracle19c-3.1
+  - name: failed_login_attempts
+    description: 'Number of failed login attemps'
+    type: Numeric
+    <% if ENV['BASELINE'] == 'High' %>
+    value: 3
+    <% elsif ENV['BASELINE'] == 'Moderate' || ENV['BASELINE'].nil? || ENV['BASELINE'] == 'Low' %> # Default
+    value: 5
+    <% end %>
+    profile: oracle-database-19c-cis-baseline
+
+# oracle19c-3.3
+  - name: password_life_time
+    description:
+    type: Numeric
+    value: 60
+    profile: oracle-database-19c-cis-baseline
```

Comment:
An ERB condition is required for HVA before setting 60 as the value. No `elsif` required in this case as HML will get a caveat